### ISSUE TITLE
The file receive method overwrite issue

### ISSUE THREAD

**Issue by ghorbani-m** [functionland/react-native-fula#4]
The `file.receive` method gets the file from the Box and stores it on the phone's storage.
If the file still exists on the phone the file will be overwritten and it causes unwanted manipulation of the user files, and in some circumstances, the user lost the original file.

## Reproduce issue

1. Send a file to the box
2. Edit the file and save it with the same name
3. Receive file from the box 

**Issue**: The edited file will be overwritten.

## Proposal

The solution could be a temp folder to store the files and then if the app wants will move them to another place.
Also, we can add a method to clear the temp folder.

**Comment by farhoud** [comment]
@ghorbani-m 

## Proposal
For android we can replace `reactContext.getFilesDir()` this with `reactContext.getExternalCacheDir()` which is temp of android apps and leave the android to manage it.
from there you can decide what to do with it.

**Comment by ghorbani-m** [comment]
@farhoud, Exactly, [here](https://github.com/ivpusic/react-native-image-crop-picker/blob/1870e4ddbea65d5c602402d4ed4423975c576354/android/src/main/java/com/reactnative/ivpusic/imagepicker/RealPathUtil.java#L105) there is another case that used context.getCacheDir().

**Comment by farhoud** [comment]
Rescive aslo sould return meta

**Comment by farhoud** [comment]
Look like you satisfy with appdir

**Comment by farhoud** [comment]
> @farhoud, Exactly, [here](https://github.com/ivpusic/react-native-image-crop-picker/blob/1870e4ddbea65d5c602402d4ed4423975c576354/android/src/main/java/com/reactnative/ivpusic/imagepicker/RealPathUtil.java#L105) there is another case that used context.getCacheDir().

move the link to https://github.com/functionland/react-native-fula/discussions/8
